assets/js/common.js

```diff
@@ -916,6 +975,18 @@ var hivepress = {
 										responseContainer.html('');
 									});
 								}
+
+								if (container && typeof response !== 'undefined' && response.hasOwnProperty('data') && response.data.hasOwnProperty('html')) {
+									var newContainer = $(response.data.html);
+
+									if (renderSettings.type === 'append') {
+										container.append(newContainer);
+									} else {
+										container.replaceWith(newContainer);
+									}
+
+									hivepress.initUI(newContainer);
+								}
```

Comment:
Please try to simplify and minimize the newly added logic based on this comment https://github.com/hivepress/hivepress-messages/pull/26/files#r1671428403 Tying it to a specific case where the form refreshes the same block on submit and also with an interval will be an issue if we'll need some block in another extension to just refresh itself, without a form. So these 2 renders should be separate, and use the same parameters with different combos of their values